src/components/CoursesSection.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react'
+import Cards from './Cards'
+import styles from './styles/coursesSection.module.css'
+
+function coursesSection() {
```

Comment:
- Component name should start with uppercase letter.
- this component should accept title, description, and list of courses as props.

Reply:
I fixed it, can you take a look at it?